cookbooks/aws-parallelcluster-common/resources/lustre/lustre_redhat8.rb

```
@@ -29,18 +29,32 @@
     log "FSx for Lustre is not supported in this RHEL version #{version}, supported versions are >= 8.2" do
       level :warn
     end
-  elsif node['cluster']['kernel_release'].include? "4.18.0-425.3.1.el8"
-    log "FSx for Lustre is not supported in kernel version 4.18.0-425.3.1.el8 of RHEL, please update the kernel version" do
-      level :warn
-    end
   else
     action_install_lustre
   end
 end
 
+def find_centos_minor_version
+  os_minor_version = ''
+
+  # kernel release is in the form 4.18.0-425.3.1.el8
+  kernel_patch_version = node['cluster']['kernel_release'].match(/^\d+\.\d+\.\d+-(\d+)\..*$/)
+  raise "Unable to retrieve the kernel patch version from #{node['cluster']['kernel_release']}." unless kernel_patch_version
+
+  os_minor_version = '2' if kernel_patch_version[1] >= '193'
```

We should raise an exception if os_minor version is 0 or 1 like we were doing for centos7

---

In the previous implementation we were only printing a warning message letting the installation continue. I agree that can be better to raise an exception. Changing that the behaviour.